core/src/allocation_optim.jl

```diff
@@ -321,9 +321,15 @@ function get_basin_data(
     node_id::NodeID,
 )
     (; graph, basin, level_demand) = p
+    (; vertical_flux) = basin
     (; Δt_allocation) = allocation_model
     @assert node_id.type == NodeType.Basin
-    influx = get_flow(graph, node_id, 0.0)
+    vertical_flux = get_tmp(vertical_flux, 0)
+    _, basin_idx = id_index(basin.node_id, node_id)
+    # NOTE: Instantaneous
+    influx =
```

Comment:
Is it possible to pull out this as a `get_flow` method and use it in `update_vertical_flux!` as well?